pkg/chart/chart.go

```diff
@@ -541,8 +541,10 @@ func (t *Testing) doInstall(chart *Chart) error {
 			namespace, release, releaseSelector, cleanup := t.generateInstallConfig(chart)
 			defer cleanup()
 
-			if err := t.kubectl.CreateNamespace(namespace); err != nil {
```

Comment:
I wonder if adding a check to see if the namespace already exists before creation would handle your use case without adding a new flag?

Reply:
That would be an option, too. In that case it should not try to delete the namepace at the end.

Reply:
Actually if I remember correctly "do not create or delete namespace" was exactly the intended behavior of https://github.com/helm/chart-testing/pull/59. If passing the `namespace` flag can currently cause an existing namespace to be deleted I view that as a bug to be patched.

Reply:
Yes looks like this was changed recently. https://github.com/helm/chart-testing/pull/184/files#diff-326b04dcc669c6ac56b6ea231f240fb2R584-R586

cc @unguiculus

Reply:
Correct. This should already be supported. I'll have to look into this.

Reply:
@unguiculus Thank you for looking into it. In the source code it looks like as if this feature has been removed.

Reply:
A regression was introduced due to Helm 3 no longer creating namespaces. This caused namespaces to always be created. #205 fixes this.

Reply:
Thanks fo looking into it. I am closing this in favor of your PR.